src/pages/Results-poll.js

```diff
@@ -45,18 +73,39 @@ function ResultsPoll() {
           {poll?.question}
         </Output>
 
-        <Output wrapperclass="outputWrapper" className="output">
+        <Output
+          wrapperclass="outputWrapper"
+          className="output"
+          backgroundcolor={{ backgroundColor: 'blue' }}
```

Comment:
In general, try to avoid setting CSS in React properties.
The properties have a structure meaning (you wouldn't set the background in HTML too.

Instead, you could set something like `answer="one"`.
In the `Output` component (I would rename it to `Answer`), you can display `blue` for `answer="one"`, `red` for `answer="two"`,...
Take a look at libraries like https://material-ui.com/components/buttons/ (`variant`) or bootstrap for more examples.